src/main/Services/TopBarMenu.js

```
@@ -4,12 +4,67 @@
 let mainWindow;
 
+const fs = require('fs');
+const path = require('path');
+const { dialog } = require('electron');
+
+// Use app-metadata.json generated at build time
+function getAppMetadata() {
+  let meta = {};
+  try {
+    let metaPath;
+    if (process.mainModule && process.mainModule.filename) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)